projects/storefrontapp/src/app/spartacus/features/cart-base-feature.module.ts

```diff
@@ -18,6 +20,18 @@ import { I18nConfig, provideConfig } from '@spartacus/core';
           module: () =>
             import('@spartacus/cart/base').then((m) => m.CartBaseModule),
         },
+        [MINI_CART]: {
+          module: () =>
+            import('@spartacus/cart/base/components/mini-cart').then(
+              (m) => m.MiniCartModule
+            ),
+        },
+        [ADD_TO_CART]: {
+          module: () =>
+            import('@spartacus/cart/base/components/add-to-cart').then(
+              (m) => m.AddToCartModule
+            ),
+        },
```

Comment:
We can't have this here without the associated schematics support.  Did havinig these in the CartBaseRootModule work?  Let's discuss.

Reply:
Associated installation schematics are now added.